master/usage/configuration/bgp.md

```diff
@@ -44,7 +46,7 @@ When creating a Calico node, you can optionally specify an AS number to use for
 the node.  If no AS number if specified, the node will use the global default
 value.
 
-Use the `calicoctl bgp default-node-as` command to set the global default AS
+Use the `calicoctl config set asNumber` command to set the global default AS
 number.  If no value is configured, the default AS number is 64511.
```

Comment:
So we decided not to fix the poorly chosen default AS number in 2.0?  This is our best opportunity to correct that.